### ISSUE TITLE
Next Series 4 targets at Griffith Uni - comments and suggestions

### ISSUE THREAD

**Issue by mcoster** [OpenSourceMalaria/Series4#40]
This is intended to be a quick and light thread to seek feedback and suggestions for the next targets for Griffith Uni students.

Two intern students from France, Pauline and Camille, will work on OSM targets in my team over the next 3 months. At the same time, GU undergrads are due to have made the first batches of the following compound in a newly introduced experiment. What are others' thoughts on the most valuable targets to make from this material, that are not actively in development elsewhere?

![gu-new-targets](https://user-images.githubusercontent.com/28282596/39455981-c35ccc4c-4d26-11e8-9980-3dd2a58e7fd6.png)

In terms of loose threads from last year, the ["Heterooxazole"](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/522), ["Heterooxadiazole"](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/521) and the ["Alkyne/alkene/alkyl"](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/520) spring to mind.

Otherwise, they could also/alternatively target other late-stage functionalized compounds. The value of this would be dependent on testing results from [this trimester's current project student compounds.](https://github.com/OpenSourceMalaria/Series4/issues/28)

Or, we could aim for LHS variations on the [poly-hydroxylated biosynthesis compound](https://github.com/OpenSourceMalaria/Series4/issues/36), although I wouldn't want to duplicate efforts with what is being done at USyd.

Any other thoughts?

**Comment by MFernflower** [comment]
IIRC @edwintse is working on the two azole compounds so my vote is for the Alkyne/alkene pair of compounds

**Comment by mcoster** [comment]
Thanks for the thoughts @david1597 - an asymmetric synthesis had certainly caught my eye. My first thoughts were either Jacobsen hydrolytic kinetic resolution of an epoxide, or Sharpless asymmetric dihydroxylation - http://doi.org/10.1055/s-1993-22547

The required substrates don't give spectacular ee's, but perhaps recryst of a crystalline derivative might help. I certainly like Sharpless AD's in terms of ease and availability of reagents.

@MFernflower - unless we can prioritise on the basis of biological hypothesis to be tested, I will have a look at your suggestions on the basis of whether we have related precursors at hand (budgetary constraints....)

@edwintse - are you currently tackling the "azoles"?

**Comment by david1597** [comment]
@mcoster Interesting...I think I have an [epoxide ](https://github.com/OpenSourceMalaria/Series4/issues/41#issue-319123302) as a side-product. The two desymmetrization papers I linked to were the ones I was following to get to the triols - I'd not yet given thought to alternative methods.

And @edwintse is making the "azoles" - he's up in the lab at it right now.

**Comment by mcoster** [comment]
OK, good to know - we won't duplicate efforts with the azoles.

@david1597 - it was really interesting to see you report that epoxide. I think the Jacobsen HKR idea is pretty speculative - I couldn't find any close precedent for a similar substrate, and an epoxide with one of the carbons being quaternary and benzylic is a worry. BTW, [this desymmetrisation of triols by silylation](https://onlinelibrary.wiley.com/doi/abs/10.1002/anie.200805338) looks interesting and includes your triol. Although I presume the silyl group would scramble under the basic conditions used for the reaction with the chlorotriazolopyrazine.

**Comment by mcoster** [comment]
Random thought just occurred to me (I might elaborate into a separate issue later..) - are there any examples of carbamate linkers to the LHS group, eg.:

![series 4 carbamates](https://user-images.githubusercontent.com/28282596/39565311-d25afda6-4efa-11e8-8f9a-1699de48d465.png)

And, if not, what are people's thoughts about them?

**Comment by david1597** [comment]
Nope, there are no examples of carbamate linkers on the LHS. Or of carbamates anywhere (aside from some BOC protected amines).

**Comment by mcoster** [comment]
Interesting.. We have a selection of aryl isocyanates on hand from another project. I wonder if we could make the LHS hydroxy compound and react it with isocyanates to make these carbamates.

**Comment by MedChemProf** [comment]
@mcoster There are also no ureas at the moment that have been synthesized and tested. The amine variant of the hydroxy-analog you mentioned did not readily react with isocyanates as expected. The scheme can be found [here](https://github.com/OpenSourceMalaria/Series4/wiki/Synthesis-of-the-Reverse-Amide--Linked-Series). I am not saying that no urea product was formed during the reaction, just that none could be isolated from a very complex reaction mixture. It has been on the To Do list, but we have not had the opportunity to revisit the reaction to attempt any optimization.

**Comment by MFernflower** [comment]
Ureas and carbamates sound like good targets! Are you going to use Curtis
rxn to get to ureas? @mcoster
